tests/test_rebase_command.rs

```diff
@@ -193,23 +193,23 @@ fn test_rebase_single_revision() {
     let stdout = test_env.jj_cmd_success(&repo_path, &["rebase", "-r", "b", "-d", "a"]);
     insta::assert_snapshot!(stdout, @r###"
     Also rebased 3 descendant commits onto parent of rebased commit
-    Working copy now at: b5ce312dacd3 (no description set)
-    Added 0 files, modified 0 files, removed 2 files
+    Working copy now at: e7299ad0c9a7 (no description set)
+    Added 0 files, modified 0 files, removed 1 files
     "###);
     insta::assert_snapshot!(get_log_output(&test_env, &repo_path), @r###"
     @ 
     o d
     o c
     | o b
-    | o a
     |/  
+    o a
```

Comment:
This seems better to me than the old behavior, though I'm not 100% sure.

Reply:
In fact, the new behavior is what the comment above this line describes as correct.